packages/@expo/router-core/src/types/metro.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * @todo Use the global type
+ * @hidden
+ */
+export interface RequireContext {
```

Comment:
note to self mostly: It's a bit unexpected to see this here. This looks unused unless it's a cast, but I was expecting a core to not have this since this would be client-only code and hence not in `@expo/router-core`?

To clarify, it's fine to be in either package, but on a first read, it stands out to me that it's moved here